core/src/main/java/org/jsmart/zerocode/core/kafka/helper/KafkaConsumerHelper.java

```diff
@@ -93,6 +99,10 @@ public static ConsumerRecords initialPollWaitingForConsumerGroupJoin(Consumer co
         throw new RuntimeException("\n********* Kafka Consumer unable to join in time *********\n");
     }
 
+    public static ConsumerRecords initialPollWaitingForConsumerGroupJoin(Consumer consumer) {
+        return initialPollWaitingForConsumerGroupJoin(consumer,null);
+    }
```

Comment:
I would suggest to remove this empty variation, as we always have the respective configuration.

Reply:
@ddingel addressed your comment